tests/interactions/environments/actuators/test_vrchat_osc_discrete_actuator.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import torch
+from vrchat_io.controller.osc import Buttons
+
+from ami.interactions.environments.actuators.vrchat_osc_discrete_actuator import (
+    VRChatOSCDiscreteActuator,
+)
+
+
+class TestVRChatOSCDiscreteActuator:
+    @pytest.fixture
+    def actuator(self) -> VRChatOSCDiscreteActuator:
```

Comment:
@yuntan すみません、やっぱりモックしてください。勝手にVRChatに値を送信されると、リセット処理がかからないなど面倒だからです。
```suggestion
    def actuator(self, mocker: MockerFixture) -> VRChatOSCDiscreteActuator:
        mocker.patch("pythonosc.SimpleUDPClient")
```